CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+- metrics-aggregates.rb: Refactored to support new named aggregates introduced in Sensu 0.24
```

Comment:
Does this break it for clients < 0.24?

Reply:
Yes, it won't work for [sensu-api 0.23](https://sensuapp.org/docs/0.23/api/aggregates-api.html) and below